src/components/LateralBar/LateralBar.tsx

```diff
@@ -1,15 +1,28 @@
-import React, {useState} from 'react';
+import React, { useState, useEffect } from 'react';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { faBars, faTimesCircle } from '@fortawesome/free-solid-svg-icons';
 import './LateralBar.scss';
-
+import { ApiService } from '../../services/ApiService';
+ 
 interface Props { onClick: any, onSelect: any }
 
-
-
-
 export const LateralBar = ({ onClick, onSelect }: Props): JSX.Element => {
-    const [device, setDevice] = useState('');
+    const [device, setDevice] = useState(ApiService.user);
+    const [deviceList, setDeviceList] = useState([]);
+
+    const handleOnClick = (target: any) => {
+        onSelect(target.value);
```

Comment:
If we don't need on select, we can remove it :)

Reply:
I realised why I need this, its to remember that we have one device selected and dont show again the 'you have not a device selected' again when you come from history. Probably it could be resolved in a better way, I will think about it